framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -525,11 +526,13 @@ private void applyDefaultsElement(final Element annotationScope, final Annotated
                 // but it doesn't work correctly and tests fail.
```

Comment:
Also, in the method's Javadoc, should "Applies default annotations to a type." be clarified to something like "Applies conservative annotations to a type", or saying something about what the default is?